components/LandingLogo/logo.module.scss

```diff
@@ -0,0 +1,6 @@
+.logo {
+  /* & img {
+    scale: 0.9;
+    transform: scale(0.9);
+  } */
+}
```

Comment:
Any particular reason to keep this class with no styles? If not then we can just remove it

Reply:
Removed it